modules/relevadRtdProvider.md

```diff
@@ -0,0 +1,99 @@
+# Relevad Real-Time Data Submodule
+
+Module Name: Relevad Rtd Provider
+Module Type: Rtd Provider
+Maintainer: anna@relevad.com
+
+# Description
+
+Relevad is a contextual semantic analytics company. Our privacy-first, cookieless contextual categorization, segmentation, and keyword generation platform is designed to help publishers and advertisers optimize targeting and increase ad inventory yield.
+
+This real-time data processing module provides quality contextual IAB categories and segments to prebid.js auction bidders. Every category and segment comes with a score measuring its relevance to the publisher’s web page. Publishers may configure their preferred minimum score setting with the module parameter “minscore”. Relevad service does not use browser cookies and is fully GDPR compliant.
```

Comment:
Might be helpful in this section to clarify that the received cats and segs are being applied in the bid request, as well as give a format to expect

Reply:
Added description
Thank you!